dev/com.ibm.ws.jdbc/resources/com/ibm/ws/rsadapter/resources/IBMDataStoreAdapterNLS.nlsprops

```diff
@@ -245,6 +245,11 @@ DSRA4004.onconnect.sql=DSRA4004E: Unable to execute onConnect SQL command \"{0}\
 DSRA4004.onconnect.sql.explanation=An error occurred while attempting to execute the specified SQL command.
 DSRA4004.onconnect.sql.useraction=Refer to the exception message to help determine the cause of the problem.
 
+# {0} name of the DataSource
+DSRA4005.invalid.logintimeout=DSRA4005E: The Application Server does not support the loginTimeout property specified in the URL or properties of the {0} DataSource.
+DSRA4005.invalid.logintimeout.explanation=The loginTimeout property is not supported when specified for a DataSource that uses the Driver type because the Application Server is unable to call getLoginTimeout on the Driver class.
+DSRA4005.invalid.logintimeout.useraction=Modify the DataSource configuration to use a DataSource type rather than a Driver type or remove the loginTimeout property.
```

Comment:
DSRA4005.invalid.logintimeout.useraction=Modify the DataSource configuration to use a DataSource type instead of a Driver type, or remove the loginTimeout property.